packages/vite/types/internal/cssPreprocessorOptions.d.ts

```diff
@@ -1,16 +1,29 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
 
 // @ts-ignore `sass` may not be installed
-import type Sass from 'sass'
+import type DartSass from 'sass'
+// @ts-ignore `sass-embedded` may not be installed
+import type SassEmbedded from 'sass-embedded'
 // @ts-ignore `less` may not be installed
 import type Less from 'less'
 // @ts-ignore `less` may not be installed
 import type Stylus from 'stylus'
 
 /* eslint-enable @typescript-eslint/ban-ts-comment */
 
+// https://github.com/type-challenges/type-challenges/issues/29285
+type IsAny<T> = boolean extends (T extends never ? true : false) ? true : false
```

Comment:
Interestingly, the mosted 👍 ed answer (https://github.com/type-challenges/type-challenges/issues/232) didn't work in our case.